src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/_help.py

```diff
@@ -16,19 +16,19 @@
     """
 helps['sql db copy'] = """
     type: command
-    short-summary: Create a copy of a database.
+    short-summary: Creates a copy of a database.
```

Comment:
@sptramer for review on these.

Reply:
For consistency, verbs should all be singular and imperative. Changes not related to the new command should be reverted.